pxf/build.gradle

```diff
@@ -122,6 +124,42 @@ subprojects { subProject ->
 }
 
 project('pxf-service') {
+
+
+    task generateSources {
```

Comment:
can you please add a comment in the code explaining the basic idea here? If I understand correctly, we replace any mention of @pxfProtocolVersion@ with the value from gradle.properties?